ui/admin/tests/helpers/selectors.js

```diff
@@ -39,3 +39,11 @@ export const DIALOG_UNSAVED_CHANGES_DISCARD =
   '.rose-dialog-footer button:first-child';
 export const DIALOG_UNSAVED_CHANGES_CANCEL =
   '.rose-dialog-footer button:last-child';
+
+export const MODAL_WARNING = 'dialog';
+export const MODAL_WARNING_CONFIRM_BTN =
```

Comment:
I see we have some selectors MODAL other DIALOG, aren't these aiming to select the same dialog element?

Reply:
You are correct, there are vars selecting the same elements because we are migrating to Hds so this file will need to be cleaned up either during/after the Hds migration.

Reply:
@laurenolivia I am still seeing `DIALOG` and `MODAL` selectors mixed

Reply:
@calcaide Can you clarify what you mean by this? Are you referring to the how the vars are named? Can you give an example of what you'd like to see different?

Are you referring to how I moved the following vars into this file, but did not rename them?
```
DIALOG_TITLE_SELECTOR
DIALOG_MESSAGE_SELECTOR
```

Reply:
I am meaning what I started this conversatio with: I see we have some selectors MODAL other DIALOG, aren't these aiming to select the same modal?

Reply:
Gotcha. The original question was pointing to [these vars](https://github.com/hashicorp/boundary-ui/blob/main/ui/admin/tests/helpers/selectors.js#L37-L41) currently in main that were left in the selectors file, but I have since removed them.

I believe the new request is to rename these vars, which I moved into this file from another test file. Is that correct?
```
DIALOG_TITLE_SELECTOR 
DIALOG_MESSAGE_SELECTOR
```
renamed to:

```
MODAL_WARNING_TITLE
MODAL_WARNING_MESSAGE
```